pkg/controller/podautoscaler/replica_calculator.go

```diff
@@ -107,9 +107,10 @@ func (c *ReplicaCalculator) GetResourceReplicas(ctx context.Context, currentRepl
 
 	if len(missingPods) > 0 {
 		if usageRatio < 1.0 {
-			// on a scale-down, treat missing pods as using 100% of the resource request
+			// on a scale-down, treat missing pods as using 100% (all) of the resource request
```

Comment:
How about `(...) as using 100% (all) of the resource request or target utilization for targets higher than 100%`?

Reply:
Addressed